WcaOnRails/app/controllers/persons_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PersonsController < ApplicationController
+  def index
+    @regions = { 'Continent' => Continent.all.map { |continent| [continent.name, continent.id] },
```

Comment:
Yeah, but where would you put it? That's the way we've done this with competitions index.